openapi.yml

```diff
@@ -84,17 +91,21 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BareDruid'
+                $ref: '#/components/schemas/Identifier'
         '404':
           description: Identifier not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
       parameters:
         - name: id
           in: path
           description: druid to return
           required: true
           schema:
             $ref: '#/components/schemas/BareDruid'
-  /suri2/namespaces/druids/identifiers:
+  /suri2/namespaces/druid/identifiers:
```

Comment:
This was what was causing the Hydrus build to fail, @ndushay, if you were interested. I added a bunch of request specs to this branch to catch these more easily. (This is part of why we've been moving away from controller specs towards request specs!)